src/ladder/Game.java

```diff
@@ -2,9 +2,28 @@
 
 public class Game {
 
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+    private Ladder ladder;
+
     public static void main(String[] args) {
-        GameManager gameManager = new GameManager();
-        gameManager.gameStart();
+        Game game = new Game();
+        game.start();
```

Comment:
```suggestion
        game.play();
```
문득 든 생각인데 이것도 괜찮지 않나요?